modules/apps/portal/portal-upgrade-test/src/testIntegration/java/com/liferay/portal/upgrade/test/ReleaseManagerOSGiCommandsTest.java

```diff
@@ -79,10 +79,10 @@ UpgradeStep.class, new DummyUpgradeStep(),
 	}
 
 	@Inject(
-		filter = "component.name=com.liferay.portal.upgrade.internal.release.ReleaseManagerImpl",
+		filter = "component.name=com.liferay.portal.upgrade.internal.release.ReleaseManagerHelper",
```

Comment:
Hey @regiCesar21, fix the package here. Thx

Reply:
fixed